content/docs/technical_practices/azure/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+# Page title as it appears in the navigation menu
+title: "Azure"
+# Adjust weight to reorder menu items (lower numbers appear first)
+weight: 6
+# Uncomment to hide nested pages in a collapsed menu
+# bookCollapseSection = true
+# Uncomment to hide this page from the navigation menu
+# bookHidden = false
+# Uncomment to exclude this page from the search
+# bookSearchExclude = true
+---
+
+# Azure
+
+Azure is Microsoft's cloud service. The Turing has a grant from Microsoft to use this.
```

Comment:
this isn't true any longer - needs review

Reply:
True, this statement is out of date. I guess we could drop the second sentence or go for something like

```suggestion
Azure is Microsoft's cloud platform, through which Turing members have access to a wide range of computing and data storage resources.
```

or 

```suggestion
Azure is Microsoft's cloud computing platform. The Turing has a contract with Microsoft which provides access to everyone with a Turing AD account.
```

depending on what we are trying to convey.